### PR TITLE
feat(ellipsis): support expand

### PR DESCRIPTION
**Pull request by chouchouji**
## Checklist

List of tasks you have already done and plan to do.

- [x] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

Describe your modifications here.

## Issues

The issues you want to close, formatted as close #1.

## Related Links

Links related to this pr.

### CODE REVIEW

File: packages/varlet-ui/src/ellipsis/Ellipsis.vue [hunk 1]
```diff
@@ -64,8 +56,17 @@ export default defineComponent({
       }
 
       expanding.value = !expanding.value
+      call(props['onUpdate:expand'], expanding.value)
```
Comment (haoziqaq) [inline packages/varlet-ui/src/ellipsis/Ellipsis.vue:59]: replaced with useVModel

File: packages/varlet-ui/types/ellipsis.d.ts [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 export declare const ellipsisProps: Record<string, any>
 
 export type EllipsisExpandTrigger = 'click'
 
 export interface EllipsisProps extends BasicAttributes {
+  expand?: boolean
   expandTrigger?: EllipsisExpandTrigger
   lineClamp?: string | number
   tooltip?: boolean | TooltipProps
+  'onUpdate:show'?: ListenerProp<(expand: boolean) => void>
```
Comment (haoziqaq) [inline packages/varlet-ui/types/ellipsis.d.ts:13]: typo error

Comment (chouchouji) [reply packages/varlet-ui/types/ellipsis.d.ts:13]: OK

File: packages/varlet-ui/src/ellipsis/Ellipsis.vue (hunk 1 above)
Comment (chouchouji) [reply packages/varlet-ui/src/ellipsis/Ellipsis.vue:59]: OK, I forget it.

**Pull request merged**